src/chrome/content/rules/Justia.com.xml

```diff
@@ -0,0 +1,158 @@
+<ruleset name="Justia.com (partial)">
```

Comment:
Would you add a comment that this domain has a wildcard DNS records and uses a wildcard certificate? https://dns.google.com/query?name=*.justia.com&type=A&dnssec=true

Reply:
@cschanaj Done.